src/Widgets/SectionWidget/SectionWidgetComponent.tsx

```diff
@@ -50,23 +50,46 @@ const ImageOrVideo = connect(function ImageOrVideo({
   const classNames = ['img-background']
   if (widget.get('backgroundAnimateOnHover')) classNames.push('img-zoom')
 
-  return (
-    // Check is a working around for issue #4767
-    // TODO: remove work around
-    background.contentType().startsWith('video/') &&
-      background.contentUrl().startsWith('https://') ? (
-      <video className={classNames.join(' ')} autoPlay loop muted playsInline>
-        <source src={background.contentUrl()} type={background.contentType()} />
-      </video>
-    ) : (
-      <InPlaceEditingOff>
-        <ImageTag
-          content={widget}
-          attribute="backgroundImage"
-          className={classNames.join(' ')}
-          alt=""
-        />
-      </InPlaceEditingOff>
+  if (background.contentType().startsWith('video/')) {
+    return (
+      <Video
+        className={classNames.join(' ')}
+        contentUrl={background.contentUrl()}
+        contentType={background.contentType()}
+        key={[
+          'SectionWidget',
+          widget.id(),
+          'Video',
+          background.contentUrl(),
+          background.contentType(),
+        ].join('-')} // Ensure the video is reloaded when the content changes
+      />
```

Comment:
In the VideoWidgetComponent we use a more straightforward approach:
```tsx
key={`${src}${posterUrl}`}
```
Here we don't even have a `posterUrl`, and `contentType` will usually only change if `src` also changes, so how about:
```tsx
<video className={className} autoPlay loop muted playsInline key={src}>
  <source src={src} type={contentType} />
</video>
```
* [ ] Try to simplify

Reply:
I just clarified my own confusion. So far I was not sure, if a `key` is global in react or not. But they are _not_ global:

> ### Note
>
> Remember that keys are not globally unique. They only specify the position _within the parent_.

Source: https://react.dev/learn/preserving-and-resetting-state#option-2-resetting-state-with-a-key